src/init.cpp

```diff
@@ -644,7 +644,7 @@ void SetupServerArgs(ArgsManager& argsman, bool can_listen_ipc)
     argsman.AddArg("-whitelistrelay", strprintf("Add 'relay' permission to whitelisted peers with default permissions. This will accept relayed transactions even when not relaying transactions (default: %d)", DEFAULT_WHITELISTRELAY), ArgsManager::ALLOW_ANY, OptionsCategory::NODE_RELAY);
 
 
-    argsman.AddArg("-blockmaxweight=<n>", strprintf("Set maximum BIP141 block weight (default: %d)", DEFAULT_BLOCK_MAX_WEIGHT), ArgsManager::ALLOW_ANY, OptionsCategory::BLOCK_CREATION);
+    argsman.AddArg("-blockmaxweight=<n>", strprintf("Set maximum BIP141 block weight (default: %d)", DEFAULT_BLOCK_MAX_WEIGHT), ArgsManager::ALLOW_ANY | ArgsManager::DEBUG_ONLY, OptionsCategory::BLOCK_CREATION);
```

Comment:
It seems this change ought to have a release note to explain why the option is no longer available by default and how to enable it.

Reply:
The option is available by default and enabled.
It is just hidden in the help text.

Reply:
Yeah, I will mention when the notes are ready to be edited.

Thanks

Reply:
Done https://github.com/bitcoin-core/bitcoin-devwiki/wiki/v30.0-Release-Notes-Draft/3a0ad5b7901c244976eeffb034305614efb350b6